ui/src/components/markdown-textarea.tsx

```diff
@@ -469,6 +469,16 @@ export class MarkdownTextArea extends Component<
     i.simpleInsert('#');
   }
 
+  handleInsertSubscript(i: MarkdownTextArea, event: any) {
+    event.preventDefault();
+    i.simpleSurround('~');
+  }
+
+  handleInsertSuperscript(i: MarkdownTextArea, event: any) {
+    event.preventDefault();
+    i.simpleSurround('^');
+  }
```

Comment:
These two functions are unused, and need icons / buttons, look at the other handleX examples above.

Reply:
Seems working and should be fixed now. Icons taken from here: https://commons.wikimedia.org/wiki/OOUI_icons#subscript